README.md

```diff
@@ -2,24 +2,77 @@
-  2. Type `react-native` and install **React Native Tools**
-  3. For more guidance view [VS Code Extension Gallery](https://code.visualstudio.com/docs/editor/extension-gallery)
-* If you haven't already, set up React Native using the steps detailed on the React Native [getting started documentation](https://facebook.github.io/react-native/docs/getting-started.html)
-* Open your React Native project root folder in VS Code.
+<!-- TABLE OF CONTENTS -->
+
+# Table of Contents
+
+- [Getting started](#getting-started)
+- [React Native commands in the Command Palette](#react-native-commands-in-the-command-palette)
+- [Debugging React Native applications](#debugging-react-native-applications)
+  - [Hermes (Android)](#hermes-android)
+  - [iOS devices](#ios-devices)
+  - [Custom scheme for iOS apps](#custom-scheme-for-ios-apps)
+  - [Expo applications](#expo-applications)
+    - [Configuring Expo](#configuring-expo)
+  - [Windows applications](#react-native-for-windows)
+  - [TypeScript and Haul](#typescript-and-haul)
+  - [Debugger configuration properties](#debugger-configuration-properties)
+- [Customization](#customization)
+  - [Logging](#logging)
+  - [Build APK and generate bundle](#build-apk-and-generate-bundle)
+  - [Specifying custom arguments for `react-native run-*` command](#specifying-custom-arguments-for-react-native-run--command)
+  - [Setting up the react-native packager](#setting-up-the-react-native-packager)
+  - [Change project root](#change-project-root)
+- [Developing inside a Docker Container](#developing-inside-a-docker-container)
+- [Contributing](#contributing)
+- [Known Issues](#known-issues)
+
+# Getting started
+
+Before going any further make sure that you:
+
+* [have a working React Native environment](https://reactnative.dev/docs/environment-setup).
+* are using [VS Code](https://code.visualstudio.com) and have [installed this extension from the Marketplace](https://marketplace.visualstudio.com/items?itemName=msjsdiag.vscode-react-native).
+* have your React Native project root folder open in VS Code.
 
 Please notice that the extension uses `.vscode/.react` directory at the project root to store intermediate files required for debugging. Although these files usually get removed after debug session ends, you may want to add this directory to your project's `.gitignore` file.
 
-## Debugging React Native applications
+# React Native commands in the Command Palette
+
+In the Command Palette, type `React Native` and choose a command.
+
+![React Native commands](images/command-palette.png)
+
+The **Run Android** command triggers `react-native run-android` and starts your app for Android.
+
+The **Run iOS** command similarly triggers `react-native run-ios` and starts your app in the iOS simulator (iPhone 6).
+
+The **Packager** commands allow you to start/stop the [**Metro Bundler**](https://github.com/facebook/metro-bundler) (formerly React Packager).
+
+The full list of commands is:
+
+|Name|Description|
```

Comment:
With that representation that table is quite poor. Command names just duplicate the descriptions. I think it could be useful if we expand descriptions a bit.

Reply:
I agree, but maybe not a blocker for this PR?
I'll open a separate issue to track this.

Reply:
I've opened #1246

Reply:
Of course, I just mentioned this to make sure that we will do it in the future.